treefunc.js

```diff
@@ -0,0 +1,14 @@
+function drawTree(h) {
+  for(var i=0; i<=h; i++){
+    var star = '';
+    for(var k = 1; k <= h - i; k++){
```

Comment:
avoid calculations like  h - i in loop, you should have stored this in variable instead